src/main/resources/META-INF/spring/ckeditor.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans-3.0.xsd">
+
+    <!-- Add JQuery in Jahia <head></head> -->
+    <bean class="org.jahia.ajax.gwt.helper.ModuleGWTResources">
+        <property name="javascriptResources">
+            <list>
+                <value>/modules/ckeditor/javascript/jquery-3.5.1.min.js</value>
```

Comment:
please use jquery module

Reply:
I hesitated to use jQuery module, do you think I have to do a dependance on jquery module, using: 
<jahia-depends>jquery</jahia-depends>
If I use the jquery js here ?

Reply:
assets module already depends on jquery , so you can be sure it's available, and it's probably ok to depend on jquery module. However adding dependencies can trigger a lot of transitive refreshes and restarts if the jquery module is updated, so it's maybe safer to not add the dependency and just set the link to /modules/jquery

Reply:
Ok I changed the link, agree that no dependency is safer for refresh, so I didnt add one.